libwild/src/args.rs

```diff
@@ -143,6 +143,8 @@ const SILENTLY_IGNORED_FLAGS: &[&str] = &[
     // so perhaps ignoring these is the right thing to do.
     "start-group",
     "end-group",
+    "(",
+    ")",
```

Comment:
I'm not sure about these two. I debugged at least one package years ago that used this variant, but they are so rare I'm not sure if there is a point even in having them here.